nms/packages/magmalte/package.json

```diff
@@ -26,7 +26,7 @@
     "@babel/register": "^7.13.16",
     "@babel/runtime": "^7.14.0",
     "@date-io/moment": "1.3.13",
-    "@fbcnms/alarms": "^0.3.1",
+    "@fbcnms/alarms": "^0.1.0",
```

Comment:
Why is this being downgraded?

Reply:
You caught a typo. Thank you. This should have been upgraded from 0.3.1 to 1.0.0.